bot/exts/halloween/candy_collection.py

```diff
@@ -179,16 +171,12 @@ async def send_spook_msg(self, author: discord.Member, channel: discord.TextChan
     def save_to_json(self) -> None:
         """Save JSON to a local file."""
         with open(json_location, 'w', encoding="utf8") as outfile:
-            json.dump(self.candy_json, outfile)
+            json.dump(dict(records=self.candy_records), outfile)
 
     @in_month(Month.OCTOBER)
     @commands.command()
     async def candy(self, ctx: commands.Context) -> None:
         """Get the candy leaderboard and save to JSON."""
-        # Use run_in_executor to prevent blocking
-        thing = functools.partial(self.save_to_json)
-        await self.bot.loop.run_in_executor(None, thing)
-
         emoji = (
```

Comment:
this emoji variable can be merged with the `EMOJIS` variable which I mentioned earlier in the review.

Reply:
Not sure how that will work out? Please take a look at [line 192](https://github.com/python-discord/seasonalbot/pull/496/files/13f7bec8e5bb0b284d30844a81789fa7c9d0e7fe#diff-22f1e6da8a1aa4597843c84ca630d90dR192) which utilizes indices of this tuple.

Applied all other suggestions on a local branch, will commit and push it all together once this gets answered.

Reply:
You can store it in a list
```python
EMOJIS = {
    "medals": (
        '\N{FIRST PLACE MEDAL}',
        '\N{SECOND PLACE MEDAL}',
        '\N{SPORTS MEDAL}'
        )
}

```